specification/operationalinsights/data-plane/Microsoft.OperationalInsights/preview/2021-05-19_Preview/OperationalInsights.json

```diff
@@ -123,6 +123,40 @@
         }
       }
     },
+    "/{resourceUri}/query": {
+      "post": {
+        "operationId": "Query_Execute",
+        "summary": "Execute an Analytics query using resource URI",
+        "description": "Executes an Analytics query for data. [Here](https://dev.loganalytics.io/documentation/Using-the-API) is an example for using POST with an Analytics query.",
```

Comment:
The dev.loganalytics.io portal was retired. The docs have moved to https://docs.microsoft.com/en-us/azure/azure-monitor/logs/api/overview.